pkgs/development/python-modules/pyobjc-framework-CoreBluetooth/default.nix

```
@@ -0,0 +1,66 @@
+{
+  buildPythonPackage,
+  darwin,
+  fetchFromGitHub,
+  lib,
+  pyobjc-core,
+  pyobjc-framework-Cocoa,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pyobjc-framework-CoreBluetooth";
+  version = "11.1";
```

Should we just do
```suggestion
  inherit (pyobjc-core) version src;
  patches = pyobjc-core.patches or [ ];
```
?